packages/trader/package.json

```diff
@@ -130,7 +130,7 @@
     "react-content-loader": "^4.2.1",
     "react-dom": "~16.8.6",
     "react-i18next": "^10.11.1",
-    "react-pose": "^4.0.8",
+    "react-pose": "^4.0.10",
```

Comment:
do we still want to update this even though it doesn't solve our issue?

Reply:
@negarn I think we should always have a default of updating whenever possible, do we not want that?

Reply:
I'm worried it might break things that are working now, that's the main reason we don't update, but sure, let's go for it